db/migrate/20160621204706_rename_stil_id_on_user.rb

```diff
@@ -0,0 +1,5 @@
+class RenameStilIdOnUser < ActiveRecord::Migration
+  def change
+    rename_column(:users, :stil_id, :student_id)
```

Comment:
> Hm, LUCAT is only needed for accesses so far.
> For election and stuff stil-id has mostly been used because it always had?
> It was a way to email the right person and such.

According to the hints, the users are supposed to enter their LUCAT-id in this field? If you want to keep stil-id, you should create a new column for LUCAT